elastic/kibana
# [Uptime] Change page headlines to use h1 element

## Summary

Resolves https://github.com/elastic/kibana/issues/40582.

It has been noted that we should be using `<h1 />` elements in our page headlines. This PR updates the UI code to rectify this.

## Testing this PR
0. Run this patch with an ES snapshot and a heartbeat instance. Navigate to Uptime plugin.
1. `Inspect` the _Overview Page_'s title.
2. Click one of the monitor names to navigate to the _Monitor Page_.
3. `Inspect` the _Monitor Page_'s title.

In both instances, the text should now be nested in `<h1 />` tags.
